packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/TimelineHeaderRow/TimelineCollapser.tsx

```diff
@@ -52,16 +53,16 @@ export default class TimelineCollapser extends React.PureComponent<CollapserProp
           />
         </Tooltip>
         <Tooltip title={getTitle('Collapse +1')} getPopupContainer={this.getContainer}>
-          <IoChevronForward
+          <IoChevronDown
             onClick={onCollapseOne}
             className="TimelineCollapser--btn TimelineCollapser--btn-size"
           />
         </Tooltip>
         <Tooltip title={getTitle('Expand All')} getPopupContainer={this.getContainer}>
-          <DoubleRightOutlined onClick={onExpandAll} className="TimelineCollapser--btn-expand" />
+          <ChevronsDown onClick={onExpandAll} className="TimelineCollapser--btn-expand" />
```

Comment:
why is down equivalent to dbl-right? We can reuse ChevronsDown but rotate by 90deg, but I don't see this in the css